pkg/controllers/management/clusterdeploy/clusterdeploy.go

```diff
@@ -239,7 +239,7 @@ func redeployAgent(cluster *v3.Cluster, desiredAgent, desiredAuth string, desire
 	}
 
 	if !reflect.DeepEqual(cluster.Spec.AgentEnvVars, cluster.Status.AppliedAgentEnvVars) {
-		logrus.Infof("clusterDeploy: redeployAgent: redeploy Rancher agents due to agent env vars mismatched for [%s], was [%v] and will be [%v]", cluster.Name, cluster.Spec.AgentEnvVars, cluster.Status.AppliedAgentEnvVars)
+		logrus.Infof("clusterDeploy: redeployAgent: redeploy Rancher agents due to agent env vars mismatched for [%s], was [%v] and will be [%v]", cluster.Name, cluster.Status.AppliedAgentEnvVars, cluster.Spec.AgentEnvVars)
```

Comment:
Nice catch swapping these 2.
On a side note, I'm wondering if there is a chance for any of these env variables to contain any sensitive information such as passwords? We only allow sensitive data to be present at trace level (FYI - https://rancher.com/docs/rancher/v2.6/en/troubleshooting/logging/)